lte/gateway/c/core/oai/test/lib/test_3gpp.cpp

```diff
@@ -329,4 +330,4 @@ int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
   OAILOG_INIT("MME", OAILOG_LEVEL_DEBUG, MAX_LOG_PROTOS);
   return RUN_ALL_TESTS();
-}
\ No newline at end of file
+}
```

Comment:
Check whether new line required or not

Reply:
This is a formatting change only. Somehow as part of clang formatting this got removed and added